### PR TITLE
Store past crash regressions in corpus backup archive.

### PR DESCRIPTION
**Pull request by inferno-chromium**
See https://github.com/google/clusterfuzz/pull/1869#discussion_r447182042

### CODE REVIEW

File: src/python/bot/tasks/corpus_pruning_task.py [hunk 1]
```diff
@@ -646,9 +646,18 @@ def do_corpus_pruning(context, last_execution_failed, revision):
   context.sync_to_gcs()
 
   # Create corpus backup.
+  # Temporarily copy the past crash regressions folder into the minimized corpus
+  # so that corpus backup archive can have both.
+  regressions_input_dir = os.path.join(context.initial_corpus_path,
+                                       'regressions')
+  regressions_output_dir = os.path.join(context.minimized_corpus_path,
```
Comment (Dor1s) [inline src/python/bot/tasks/corpus_pruning_task.py:653]: so this would be non-minimized, but we don't expect a lot of files in there?

Comment (inferno-chromium) [reply src/python/bot/tasks/corpus_pruning_task.py:653]: i dont think we can minimize crash regressions as the input itself is important for regression testing. over time, size can get large, but this should be only for crashes with bugs attached, so number will be very low. over time, i can think of removing maybe ooms and timeout from being archived if needed, but it feels weird to exclude them right now, i think a project maintainer cares about all regressions.

Comment (Dor1s) [reply src/python/bot/tasks/corpus_pruning_task.py:653]: makes sense! Should be fine since it's only the ones with bugs attached. Thanks!

**Pull request merged**